### PR TITLE
Feature/tags

### PR DESCRIPTION
**Pull request by sisitrs2**
Added tags for server. 
This could be used for any purpose, signaling the OS or AV were in mind during the development.

### CODE REVIEW

File: static/css/servers.css [hunk 1]
```diff
@@ -502,4 +521,71 @@ input[type="file"] {
     overflow-wrap: break-word;
     min-height: 200px;
     max-height: 500px;
-}
\ No newline at end of file
+}
+
+#tag-box {
```
Comment (Sh3lldor) [inline static/css/servers.css:526]: Maybe we should make the box bigger

File: static/css/servers.css [hunk 2]
```diff
@@ -502,4 +521,71 @@ input[type="file"] {
...
+#tag-box input {
+    margin: 20px;
+    text-align: center;
+}
+
+.checkbox-container input {
+    display: none;
+}
+  
+.checkbox-container .checkbox-indicator {
```
Comment (Sh3lldor) [inline static/css/servers.css:561]: border-radius maybe?

**Review by Sh3lldor — CHANGES_REQUESTED** [review]

File: static/css/servers.css (hunk 1 above)
Comment (sisitrs2) [reply static/css/servers.css:526]: Done

File: static/css/servers.css (hunk 2 above)
Comment (sisitrs2) [reply static/css/servers.css:561]: Done

**Review by Sh3lldor — APPROVED** [review]
LGTM!

**Pull request merged**